### PR TITLE
First attempt at CSS modules

### PR DESCRIPTION
**Pull request by wbobeirne**
This PR serves as a set of guidelines around future styling, and two examples by way of converting the the Header and the Footer. The Header converted quite nicely, but the Footer is a good example of how tough some components can be given how rooted they are in bootstrappery.

The tl;dr of how I'm proposing we style is to have an `scss` file per `jsx` file (that needs styling) in the same directory, and for the `jsx` file to `import './FileName.scss';`. Styling should adhere to SuitCSS naming conventions to prevent collisions, and promote lightweight selectors using the `&` selector. The README is much more thorough in explaining this.

The main gnarly thing in this PR is just a straight copy / paste / convert of `etherwallet-variables.less` to `common/sass/variables.scss` for continuity. We could ideally trim down all the bootstrap-centric variables, and any unused ones, but I wanted to reduce friction for people converting styles.

Let me know what y'all think.

### CODE REVIEW

**Comment by wbobeirne** [comment]
cc @tayvano @zerko

File: webpack_config/webpack.base.js [hunk 1]
```diff
@@ -20,19 +20,9 @@ module.exports = {
...
-    alias: {
-      actions: `${config.srcPath}/actions/`,
-      api: `${config.srcPath}/api/`,
-      reducers: `${config.srcPath}/reducers/`,
-      components: `${config.srcPath}/components/`,
-      containers: `${config.srcPath}/containers/`,
-      styles: `${config.srcPath}/styles/`,
-      less_vars: `${config.srcPath}/styles/etherwallet-variables.less`
-    },
-    // FIXME why aliases then?
```
Comment (wbobeirne) [inline webpack_config/webpack.base.js]: Why indeed? Removing them seemed to work fine, and I was in this file figuring out SASS resolves anyway, so I nix'd em.

**Comment by dternyak** [comment]
@tayvano Approved over slack.

**Comment by dternyak** [comment]
Thank you!

**Pull request merged**